addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -457,6 +457,8 @@ $o-color-extras-nesting-selector: '&, .o_colored_level &';
         'o-color-3': change-color(#F5F0F0, $hue: hue($-primary), $saturation: min(saturation($-primary), saturation(#F5F0F0))),
         'o-color-4': #FFFFFF,
         'o-color-5': change-color(#2e1414, $hue: hue($-primary), $saturation: min(saturation($-primary), saturation(#2e1414))),
+        'o-color-btn-primary-text': color-contrast($-primary, #212529, #FFFFFF, 3, #FFFFFF),
+        'o-color-btn-secondary-text': color-contrast($-o-color-2, #212529, #FFFFFF, 3, #FFFFFF),
```

Comment:
You are in the function which creates the color palettes, you should not have this here. First, a palette could be created without using this function. You should rather update where we "print" CSS variable in the DOM.

Reply:
The issue here is that "color-contrast" is a scss function, so i have no access to this function afterwards. Do you have a solution or a clue maybe?

Reply:
> so i have no access to this function afterwards

You have access to that function is all SCSS files.

> Do you have a solution or a clue maybe?

=> You should rather update where we "print" CSS variable in the DOM. = check `print-variable`, we already "print" many values to be usable in the editor. This is probably about doing the same thing for this :+1: